media/css/firefox/channel.less

```diff
@@ -4,195 +4,669 @@
+                .inline-block;
+                margin: 0;
+                padding: @baseLine 0;
+
+                &:after {
+                    color: #56565a;
+                    content: '|';
+                    margin: 0 10px;
+                }
+
+                &:last-child:after {
+                    content: none;
+                }
+            }
+        }
+
+
+        a:link,
+        a:visited {
+            color: @linkBlue;
+        }
+
+        a:hover,
+        a:active,
+        a:focus {
+            color: @linkBlueHover;
+        }
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+        header {
+            min-height: 160px;
+            padding-left: 180px;
+            width: 270px;
+
+            h2 {
+                .font-size(30px);
+            }
+
+            p {
+                .font-size(24px);
+            }
+        }
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        header {
+            background-position: top center;
+            float: none;
+            min-height: 0;
+            padding: 160px 0 0;
+            text-align: center;
+            width: auto;
+        }
+
+        .download-button,
+        .testflight-cta {
+            float: none;
+            text-align: center;
+        }
+
+        .notice {
+            .font-size(@baseFontSize);
+        }
+
+        footer ul {
+            padding-bottom: @baseLine;
+
+            li {
+                .font-size(@largeFontSize);
+                display: block;
+                padding: @baseLine 0 0;
+
+                &:after {
+                    content: none;
+                    margin: 0;
+                }
+            }
+        }
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+
+        header {
+            padding: 115px 0 0;
+
+            h2 {
+                .font-size(26px);
+            }
+
+            p {
+                .font-size(22px);
+            }
+        }
+
+        .notice,
+        footer ul li {
+            .font-size(@baseFontSize);
+        }
+    }
 }
 
-#main-content {
-    background: url(/media/img/firefox/channel/carousel-bg.png) 50% 70px no-repeat;
+.html-rtl .channel {
+
+    header {
+        background-position: top right;
+        float: right;
+        padding-left: 0;
+        padding-right: 240px;
+        text-align: right;
+    }
+
+    .download-button,
+    .testflight-cta {
+        float: left;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+
+        header {
+            background-position: top center;
+            float: none;
+            padding-right: 0;
+            text-align: center;
+        }
+
+        .download-button,
+        .testflight-cta {
+            float: none;
+        }
+    }
 }
 
-.download-box {
-    .inline-block;
-    margin: 0 (@gridGutterWidth / 2);
-    text-align: left;
+#beta header {
+    .at2x('/media/img/firefox/template/beta-logo.png', 180px, 189px);
```

Comment:
It looks like there is only the homepage and `/desktop/tips` pages using the common directory. This change is already large enough however, so I think this would make a good first bug 👍